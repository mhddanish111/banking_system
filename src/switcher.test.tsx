import { shallow } from 'enzyme';
import Switcher from './switcher';

describe('<Switcher />', () => {
  const wrapped = shallow(<Switcher />);
  it('renders Switcher Component', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
