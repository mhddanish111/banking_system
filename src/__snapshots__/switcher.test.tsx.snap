// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Switcher /> renders Switcher Component 1`] = `
<Suspense
  fallback={<div />}
>
  <BrowserRouter>
    <Routes>
      <Route
        element={<UNDEFINED />}
        path="/"
      />
      <Route
        element={<UNDEFINED />}
        path="/sign-up"
      />
      <Route
        element={<UNDEFINED />}
        path="/dashboard"
      />
      <Route
        element={<UNDEFINED />}
        path="/account"
      />
      <Route
        element={<UNDEFINED />}
        path="/open-account"
      />
      <Route
        element={<UNDEFINED />}
        path="/credit-card"
      />
      <Route
        element={<UNDEFINED />}
        path="/credit-card-apply"
      />
      <Route
        element={<UNDEFINED />}
        path="/profile"
      />
      <Route
        element={<UNDEFINED />}
        path="/change-password"
      />
      <Route
        element={<UNDEFINED />}
        path="/create-account"
      />
      <Route
        element={<UNDEFINED />}
        path="*"
      />
    </Routes>
  </BrowserRouter>
</Suspense>
`;
